Migrations/Version20240726082447.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the "Shared Project Timesheets Bundle" for Kimai.
+ * All rights reserved by Fabian Vetter (https://vettersolutions.de).
+ *
+ * For the full copyright and license information, please view the LICENSE file
+ * that was distributed with this source code.
+ */
+
+namespace KimaiPlugin\SharedProjectTimesheetsBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240726082447 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add nullable customer_id column and make project_id nullable';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $tableName = Version2020120600000::SHARED_PROJECT_TIMESHEETS_TABLE_NAME;
+
+        // raw sql needed because column position is not supported
+        $this->addSql(\sprintf('ALTER TABLE %s ADD customer_id INT(11) DEFAULT NULL AFTER id', $tableName));
```

Comment:
order of columns doesn't matter. portability does.
please change this to use the schema API.

Reply:
Done. I find it strange that order is not supported. It is supported in laravel migrations. Order of columns is not unimportant but I agree portability is more important.